gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dprofilepanel_ttt2.lua

```diff
@@ -177,7 +177,8 @@ function PANEL:DrawModel(x, y, w, h)
     render.SetColorModulation(self.colColor.r / 255, self.colColor.g / 255, self.colColor.b / 255)
     render.SetBlend((self:GetAlpha() / 255) * (self.colColor.a / 255))
 
-    for i = 0, 6 do
+        -- iterates over the model lighting enum: https://wiki.facepunch.com/gmod/Enums/BOX
+        for i = 0, 6 do
```

Comment:
this indentation change doesn't seem correct

Reply:
still working on this PR!